users/tests.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from . import models
+
+
+PHONE_NUM = "0528883333"
+EMAIL = "user@gmail.com"
+PASSWORD = "AdminU$er123"
+NAME = "user"
+
+VALID_PREFIX_ERROR = "{} is not a valid phone number. The number should start with 05"
+NOT_VALID_PREFIX_ENDING_ERROR = "{} is not a valid phone number. " + \
+                                "The number should start with 05 and a number that is not 7"
+NOT_SUPPORTED_ERROR = "{} is not supported"
+PHONE_NUMBER_LENGTH = "{} is not a valid phone number. The number should have {} digits"
+CHERECTERS_ERROR = "{} has at least one character"
+EMAIL_ERROR = "Email is not valid"
+
+
+@pytest.fixture
+def user0():
+    return models.User(email=EMAIL, phone_number=PHONE_NUM, password=PASSWORD, name=NAME)
+
+
+def test_new_user(user0):
+    assert user0.email == EMAIL
+    assert user0.phone_number == PHONE_NUM
+    assert user0.name == NAME
+    assert user0.profile_pic == 'profile_pic.svg'
+    assert user0.password == PASSWORD
+
+
+def create_user(email, phone_number, password, name):
+    return models.User(email=email, phone_number=phone_number, password=password, name=name)
+
+
+@pytest.mark.parametrize("users, excpected_error", [
+    (create_user(EMAIL, "090398", PASSWORD, NAME), VALID_PREFIX_ERROR.format("090398")),
```

Comment:
please prevent repeating yourself like calling the create_user in every line instead of doing it once inside the method

 
```suggestion
    (create_user(EMAIL, "090398", PASSWORD, NAME), VALID_PREFIX_ERROR.format("090398")),
```

Reply:
Done